gnosis/gp-swap-ui
# 280/token list rounding

# Summary

Closes #280 

Showing lots more precision on token list dropdown balances:
![screenshot_2021-04-05_10-47-12](https://user-images.githubusercontent.com/43217/113605982-3fd2aa00-95fc-11eb-92c8-9e6d6632da25.png)


~Small change surgically inserting a `const` into original code. Trying out a different approach than the `mod` files.~
Using the `Mod` files approach


# Testing
1. Having balance in at least one token, open the token selector dropdown
2. Wait for all balances to load
3. Verify your tokens are displayed with the highest precision available

**Note:** 
- At most the 7 most relevant digits will be displayed.
- If there are more than 7, a `...` will be appended to the end
- That applies to both values below and above `0`.
